src/uwtools/drivers/facade.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument
+# remove these disables once implemented
+'''
+This file contains the scaffolding for the strangler pattern wrapper. 
+It is a python script that will be called by the user to run the 
+manage experiments. It will call the other tools in the uwtools package 
+to do the work. It will also be responsible for setting up the 
+working directory and moving files around. 
+
+See UW documentation for more information:
+https://github.com/ufs-community/workflow-tools/wiki/Implementation-of-a-Service-Oriented-Architecture-Fa%C3%A7ade
+'''
+
+import abc
+
+class Facade: # pragma: no cover
```

Comment:
I also wonder if `Facade` might be too generic a name. I don't have a better suggestion at the moment, but maybe one will emerge as we flesh this out over time.

Reply:
Extremely generic - I'm not a fan of patterns-named-after-patterns, but for lack of a better way, sometimes it reluctantly has to be shaped like it is I guess